api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/BaseDaoImpl.java

```diff
@@ -539,6 +597,33 @@ protected void handleParticipantReference(Criteria criteria, ReferenceAndListPar
 		}
 	}
 	
+	protected Optional<Criterion> handleCode(Criteria criteria, TokenAndListParam concepts, @NotNull String conceptAlias) {
```

Comment:
I don't think we need a separate method for this any more. Just inline this for `handleCodeableConcept`. However, this function *should* still take the `conceptMapAlias` and `conceptReferenceTermAlias` parameters instead of hard-coding them as `cm` and `crt`.